include/boost/utility/ostream_write.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 Glen Joseph Fernandes
+(glenjofe@gmail.com)
+
+Distributed under the Boost Software License, Version 1.0.
+(http://www.boost.org/LICENSE_1_0.txt)
+*/
+#ifndef BOOST_UTILITY_OSTREAM_WRITE_HPP
+#define BOOST_UTILITY_OSTREAM_WRITE_HPP
+
+#include <boost/config.hpp>
+#include <iosfwd>
+
+namespace boost {
+namespace detail {
+
+template<class charT, class traits>
+inline std::size_t
+osw_put(std::basic_ostream<charT, traits>& os, const charT* data,
+    std::size_t size)
+{
+    return static_cast<std::size_t>(os.rdbuf()->sputn(data, size));
+}
+
+template<class charT, class traits>
+inline bool
+osw_fill(std::basic_ostream<charT, traits>& os, std::size_t size)
+{
+    charT c = os.fill();
+    charT fill[] = { c, c, c, c, c, c, c, c };
```

Comment:
This way of initialization is not maintenance friendly. If `chunk` is changed, it's easy to forget to modify the initialization. A loop would be better.

Reply:
Nevermind, you've changed the code already.

Reply:
`chunk` shouldn't need to be changed at all, since its value is obtained from `sizeof fill / sizeof *fill`.